TestProject/ClientApp/src/components/EditPostForm.js

```diff
@@ -0,0 +1,54 @@
+﻿import React, { useState, useEffect } from 'react';
```

Comment:
Why do we need different components for create and delete? They pretty much the same.